Orange/widgets/unsupervised/owcorrespondence.py

```diff
@@ -102,6 +103,7 @@ def __init__(self):
     def set_data(self, data):
         self.closeContext()
         self.clear()
+        self.Error.no_disc_vars.clear()
```

Comment:
Since the widget is reset here, I suppose that no error should remain visible from the previous data. So you can safely call `self.Error.clear()` instead of `self.Error.no_disc_vars.clear()`, and then you can also remove `else: self.Error.empty_data.clear()` below.